scripts/sysdig-probe-loader

```diff
@@ -22,6 +22,51 @@
 # a container or in other weird environments.
 #
 
+#
+# Returns 1 if $cos_ver > $base_ver, 0 otherwise
+#
+cos_version_greater()
+{
+	if [[ $cos_ver == $base_ver ]]; then
+		return 0
+	fi
+
+	#
+	# COS build numbers are in the format x.y.z
+	#
+
+	local IFS=.
+
+	echo $cos_ver | read -r a b c
+	echo $base_ver | read -r d e f
+
+	# Test the first component
+	if [[ $a -gt $d ]]; then
+		return 1
+	elif [[ $d -gt $d ]]; then
```

Comment:
I think there's a typo on this line. Based on the other if statements, this needs to `$d -gt $a`.